esmvaltool/utils/cmorizers/obs/cmorize_obs_Landschutzer2014.py

```diff
@@ -0,0 +1,119 @@
+"""
+# #############################################################################
+# ESMValTool CMORizer for Landschutzer2014 data
+# #############################################################################
+#
+# Tier
+#    Tier 2: other freely-available dataset.
+#
+# Source
+#    ftp://ftp.nodc.noaa.gov/nodc/archive/arc0105/0160558/1.1/data/0-data/
+#
+# Last access
+#    20190227
+#
+# Download and processing instructions
+#    Download the following file in ${RAWOBS}/Tier2/Landschutzer2014:
+#     pco2_1998-2011_ETH_SOM-FFN_CDIAC_G05.nc
+#
+#
+# Modification history
+#    20190227-A_lova_to: written.
+#
+# #############################################################################
+"""
+
+import datetime
+import logging
+import os
+
+import iris
+from cf_units import Unit
+import numpy as np
+
+from .utilities import (_add_metadata,
+                        _convert_timeunits,
+                        _fix_coords,
+                        _read_cmor_config,
+                        _roll_cube_data,
+                        _save_variable)
+
+logger = logging.getLogger(__name__)
+
+# read in CMOR configuration
+cfg = _read_cmor_config('Landschutzer2014.yml')
+proj = cfg['proj']
+timestamp = datetime.datetime.utcnow()
+timestamp_format = "%Y-%m-%d %H:%M:%S"
+now_time = timestamp.strftime(timestamp_format)
+proj['metadata_attributes']['CMORcreated'] = now_time
+VAR_TO_CMOR = cfg['VAR_TO_CMOR']
```

Comment:
Just one global configuration variable, CFG, should be enough, these should be deleted

Reply:
not that straightforward since there are specific variables in specific files, the mapping needs to be done somewhere; the inclusion of this and a bunch of other global variables in the cmorizer script is not optimal BUT this type of script is the gateway of a new user to the cmorization process and the fact they are in the cmorizer script is way more straightforward for a new user to write their own script than to get lost in the more complicated `utilities`. I think @tomaslovato appreciated this and he's written his script faster this way

Reply:
renaming random stuff from CFG to other global variables makes it more difficult to understand the code

Reply:
@bouweandela Actually OBS data come with a number of different names so there is the need to map the original names to the CMOR ones that are then assigned in `extract_variable` function.

Reply:
cfg here is not the typical esmvaltool cfg, it's the OBS's configuration to map crazy raw obs stuff to standard cmor and add metadata; this prompts the user to think a bit before creating such a file (when one adds a cmorizer they have to create this cfg as well), have full access to the cfg inputs in their cmorizer script, and see things more clearly. At least that's my view on it. Dumbing down things and adding otherwise unwanted global variables can be more transparent to first time users

Reply:
@bouweandela The time assignment to `CMORcreated` was moved in utilities.py 
https://github.com/ESMValGroup/ESMValTool/blob/9aafeacb8e64a8f837de9d979e68847c76379197/esmvaltool/utils/cmorizers/obs/utilities.py#L20-L26